portfolio/src/main/webapp/gameoflife.js

```diff
@@ -0,0 +1,156 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// CODE FOR GAME OF LIFE
+
+// Page specific variables
+var canvas = document.getElementById("mycanvas");
+var ctx = canvas.getContext("2d");
+var stop = true;
+var interval, startTime, now, then, elapsed;
```

Comment:
stick to using one kind of js var. i see you use const below. then make these "let" over "var"